src/python/WMCore/WMRuntime/Tools/Scram.py

```diff
@@ -43,7 +43,9 @@
 ARCH_TO_OS = {'slc5': ['rhel6'],
               'slc6': ['rhel6'],
               'slc7': ['rhel7'],
-              'el8': ['rhel8'], 'cc8': ['rhel8'], 'cs8': ['rhel8'], 'alma8': ['rhel8']}
+              'el8': ['rhel8'], 'cc8': ['rhel8'], 'cs8': ['rhel8'], 'alma8': ['rhel8'],
+              'el9': ['rhel9'], 'cc9': ['rhel9'], 'cs9': ['rhel9'], 'alma9': ['rhel9']
```

Comment:
I will look at the unit test, thanks.

For this question, I admit that I copied the `el8` line w/o knowing if those ARCH's may exist. I am not sure those exist either. I am not sure where you are looking. E.g. I find `el9` and `cs9` here

<pre>belforte@lxplus807/~&gt; ls /cvmfs/cms.cern.ch/|grep 9_amd
cs<font color="#EF2929"><b>9_amd</b></font>64_gcc11/
el<font color="#EF2929"><b>9_amd</b></font>64_gcc11/
el<font color="#EF2929"><b>9_amd</b></font>64_gcc12/
el<font color="#EF2929"><b>9_amd</b></font>64_gcc13/
belforte@lxplus807/~&gt; 
</pre>

while for `8` same command reports `cc8` `cs8` `el8` `rhel8` `alma8` and `ubi8`. 
The latter is not mapped atm. How do we find the correct list ? 

I am happy to start with `el9` and `cs9` and add more as/if needed. Indeed there's no point in listing SCRAM_ARCH's which do not exist in SCRAM !
I do not even know if `el9` and `cs9` are different things, or all aliases for `alma9` at the end of the day, since we can only pick a running container as `rhel7|8|9`